eslint-config-internal/package.json

```diff
@@ -12,12 +12,19 @@
     "type": "git",
     "url": "git://github.com:mate-academy/fed.git"
   },
+  "_moduleAliases": {
+    "@": "./src"
+  },
   "dependencies": {
+    "module-alias": "^2.2.2",
```

Comment:
different services use different ways to resolve modules
let's keep here only config